pom.xml

```diff
@@ -55,6 +55,13 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-javadoc-plugin</artifactId>
+                <configuration>
+                    <additionalparam>-Xdoclint:none</additionalparam>
```

Comment:
To generate the javadoc html for viewing as our end users would. This makes it a lot easier to see which classes have gaps in documentation.  Java 8 is very strict and doesn't allow <p/> in a javadoc so the additionalparam turns javadoc lint off.